lexers/h/http.go

```diff
@@ -31,7 +31,28 @@ var HTTP = internal.Register(httpBodyContentTypeLexer(MustNewLexer(
 			{`.+`, EmitterFunc(httpContentBlock), nil},
 		},
 	},
-)))
+).SetAnalyser(func(text string) float32 {
+	if strings.HasPrefix(text, "GET") ||
```

Comment:
The original code not only checks for the http verbs, but also the preceeding slash. E.g. `GET /`. See https://github.com/pygments/pygments/blob/master/pygments/lexers/textfmts.py#L198

Reply:
👀